Install-Guide/Community_Packages.md

```diff
@@ -0,0 +1,36 @@
+### Community Packages
+
+An **X** means packages are planned to be in the repository.
+A **—** means we have no plans to build the version for the repository.
+**d.g.o** means packages will (also) be provided on https://download.gluster.org
+**DNF/YUM** means the packages are included in the Fedora updates or updates-testing repos.
+
+|                   |             |   3.9    |   3.8    |   3.7    |   3.6    |
+|-------------------|-------------|:--------:|:--------:|:--------:|:--------:|
+|CentOS Storage SIG¹|el5          |    —     |    —     |   d.g.o  |   d.g.o  |
```

Comment:
This numerical superscript here (`1`) and elsewhere. What do they point to or signify ?